addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/edit.js

```diff
@@ -206,6 +233,30 @@ export default class CustomSubsidyFormFieldsClimateSubsidyCostsTableEditComponen
     }
   }
 
+  @action
+  addCustomData() {
+    let uuid = uuidv4();
+    const tableEntryUri = new rdflib.NamedNode(
+      `http://data.lblod.info/id/climate-table/row-entry/${uuid}`
+    );
+
+    this.customDataEntries = [
+      ...this.customDataEntries,
+      {
+        uri: tableEntryUri,
+        uuid,
+        temp: `http://data.lblod.info/id/subsidies/rules/custom/${uuid}`,
```

Comment:
This was temporarily passed here since I based the custom-data component on the other row components and they expect this to be passed.

Do we need a business rule for the custom entries? If so, should we just make it static?